### PR TITLE
Modified cluster flag to support quotes

### PR DESCRIPTION
**Pull request by mugdha-adhav**

### CODE REVIEW

File: pkg/execute/executor.go [hunk 1]
```diff
@@ -146,8 +144,21 @@ func runKubectlCommand(args []string, clusterName string, isAuthChannel bool) st
 		}
 		if strings.HasPrefix(arg, ClusterFlag.String()) {
 			if arg == ClusterFlag.String() {
+				if index == len(args)-1 {
+					return ""
+				}
+				arg = args[index+1]
```
Comment (PrasadG193) [inline pkg/execute/executor.go]: @mugdha-adhav can we use `continue` here?

Comment (mugdha-adhav) [reply pkg/execute/executor.go]: As discussed, will add comments to make code readable.

File: pkg/execute/executor.go [hunk 2]
```diff
@@ -144,10 +142,27 @@ func runKubectlCommand(args []string, clusterName string, isAuthChannel bool) st
...
 			continue
 		}
+		// Check --cluster-name flag
 		if strings.HasPrefix(arg, ClusterFlag.String()) {
+			// Check if flag value is in current or next argument
 			if arg == ClusterFlag.String() {
+				if index == len(args)-1 {
+					return ""
+				}
+				arg = args[index+1]
```
Comment (PrasadG193) [inline pkg/execute/executor.go]: Can we avoid reassigning arg? The above `for loop` already takes care of it. Also rename `checkFlag` to something more meaningful like `isClusterFlagPresent`? `continue` here and process the value of `--cluster-name` in the next iteration.

Comment (mugdha-adhav) [reply pkg/execute/executor.go]: Made required changes and commited.

**Pull request merged**